### PR TITLE
Weather App - Completed

### PR DESCRIPTION
**Pull request by rsnazario**
This branch is designed to evaluate: 
- [x] Javascript API Access
- [x] Displaying of API received data
- [x] Async / await 
- [x] Webpack organization
- [x] Layout Provided
- [x] Javascript Best Practices

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @rsnazario 
### You did well so far :clap: However, you have a few issues to fix before the project can be approved.
- [x] Add a toggle button to change between Celsius to Fahrenheit
- [x] Use es6 functions to implement your logic, see example below:-
```
hello = () => {
  return "Hello World!";
}
```
### Make the changes and submit for another review. Good luck.
- Happy coding!

By Raymond
Slack @iamraymond

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi @rsnazario  👋

**Nice work so far on this project.👍**

However, please refer to the comments below for changes to be made.

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: src/UIController.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  function setCurrentWind(wind) {
+    document.getElementById('current-wind').innerHTML = Math.round(wind);
+  }
+
+  function setMaxMinTemps(minTemp, maxTemp) {
+    document.getElementById('min-temp').innerHTML = Math.floor(minTemp);
+    document.getElementById('max-temp').innerHTML = Math.ceil(maxTemp);
+  }
+
+  function setCurrentInformation(data) {
```
Comment (mikenath223) [inline src/UIController.js]: - [x] Kindly change this regular functions to arrow functions. Also, make this change for every module having a regular function expression.

Comment (rsnazario) [reply src/UIController.js]: I am sorry, I really prefer the 'function blabla' pattern more. I understood that I could mix patterns.

I will keep in mind for the next projects =)

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :trophy: 

Hi @rsnazario , congratulations on a great job :+1: Code is clean :slightly_smiling_face: 
However, before we can approve the project, we need to address the comment made by the previous TSE where we were told to use ES6 syntax. Kindly check the screenshot for the project requirements
sd
- We should also check for the error handling

Once these are fixed, kindly submit another code review :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Review by Dandush03 — APPROVED** [review]
# Status: :rocket: Approved :rocket: :smile: 
### Great work modeling your code to ES6 syntax, I promise you that once you start using react you will prefer using it.
# Happy Coding :nerd_face: :computer:

**Pull request merged**